src/mesh/python/mesh_types.py

```diff
@@ -19,7 +19,7 @@ def __init__(self):
         # -- required data
         self.ndim = 0  # number of dimensions
         self.num_nodes = 0  # total number of nodes
-        self.coordinates_per_node = np.array([])  # coordinate array indexed by node
+        self.noordinates_per_node = np.array([])  # coordinate array indexed by node
```

Comment:
Typo?

Reply:
Oops, nice catch!

Reply:
Would it be possible to change line 29 below to reflect the latest version of `self.nodes_per_side` (sorry - can't seem to highlight it)?

Reply:
Yep, I updated `faces_per_cell`, `nodes_per_face`, and `nodes_per_side`

Reply:
Thank you!